packages/contracts/contracts/MarketRegistry.sol

```diff
@@ -61,6 +61,8 @@ contract MarketRegistry is
 
     TellerAS public tellerAS;
 
+    mapping(uint256 => PaymentCycleType) public paymentCycleType;
```

Comment:
it actually wouldn't hurt to _append_ the `PaymentCycleType` to the `Marketplace` struct mapping instead of making a new one